### PR TITLE
Update in-tree extensions package files

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

Caused by the increase of `npm` version.

### CODE REVIEW

**Review by jakolehm — APPROVED** [review]

File: src/extensions/npm/extensions/package.json [hunk 1]
```diff
@@ -1,25 +0,0 @@
-{
```
Comment (jakolehm) [inline src/extensions/npm/extensions/package.json]: We should not remove these.

**Review by jakolehm — CHANGES_REQUESTED** [review]

File: src/extensions/npm/extensions/package.json (hunk 1 above)
Comment (Nokel81) [reply src/extensions/npm/extensions/package.json]: Yes that is true, I am finding that `make dev -B` delete these once webpack starts to be run. Are you seeing this?

**Review by jakolehm — APPROVED** [review]

**Comment by jakolehm** [comment]
Not needed because npm upgrade was reverted?

**Comment by Nokel81** [comment]
Fair, we will probably need this once we do that again

**Pull request closed without merging**